src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -19,92 +19,89 @@ import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-erro
 
 * How can I improve the performance of my infrastructure?
 * Were unique user sessions up or down last week compared to the week before?
-    </TabsPageItem>
-    </TabsPages>
+ </TabsPageItem>
+ </TabsPages>
 </Tabs>
 
-### Get help creating NRQL queries [#NRQL-console]
+### Create NRQL queries [#NRQL-console]
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool to gain insights into systems. Still, when you're in a hurry, you might not have time to create the correct query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 <img
-        title="Screenshot showing the Ask AI button in the New Relic platform"
-        alt="ask AI button"
-        src={askgrok}
-    />
-
+ title="Screenshot showing the Ask AI button in the New Relic platform"
+ alt="ask AI button"
+ src={askgrok}
+ />
 
 To enable NRAI in the NRQL console: 
 
 1. Go to the user menu at the bottom left of the New Relic UI.
-2.  Click the <DoNotTranslate>**Features**</DoNotTranslate> tab.
+2. Click the <DoNotTranslate>**Features**</DoNotTranslate> tab.
 3. Scroll down to the <DoNotTranslate>**New Relic AI**</DoNotTranslate> section and click <DoNotTranslate>**Enable**</DoNotTranslate>.
 
 ### Get help analyzing error logs [#errors]
 
-Error log lines can be difficult to parse. New Relic AI helps you learn more about errors, query for related information, and ask about next steps. 
-
-<img
-    title="explain this error"
-    alt="A screenshot depicting the explain this error feature in New Relic Logs"
-    src={aiNewRelicAiExplainThisError}
-/>
-
-<figcaption>
-    To ask our AI about an error in your logs Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> > <DoNotTranslate>**Logs**</DoNotTranslate> > <DoNotTranslate>**All logs**</DoNotTranslate> >  and search for `errors`.
-</figcaption>
+Error logs can be challenging to parse. New Relic AI helps you learn more about errors, query for related information, and ask about next steps. 
```

Comment:
I think "It's often challenging to parse error logs" would help prevent a use of passive voice.